vnext/Desktop.ABITests/MemoryTrackerTests.cpp

```diff
@@ -7,14 +7,15 @@
 #include <winrt/base.h>
 #include <winrt/facebook.react.h>
 
+#include "ActivationFactory.h"
 #include "MessageQueueShim.h"
 
+#include "ActivationFactory.h"
```

Comment:
The file seems to be included twice.

Reply:
Thanks for the catch, fixed.

---
In reply to: [423289819](https://github.com/microsoft/react-native-windows/pull/4848#discussion_r423289819) [](ancestors = 423289819)